README.md

```diff
@@ -56,7 +56,9 @@ You can run this tutorial on your Raspberry Pi, or adapt the steps for a regular
 
 Automate everything within < 60 seconds and get a public URL and IP address back. Customise as required, or adapt to your preferred cloud such as AWS EC2.
 
-* [Provision faasd 0.7.5 on DigitalOcean with Terraform 0.12.0](https://gist.github.com/alexellis/fd618bd2f957eb08c44d086ef2fc3906)
```

Comment:
I meant not to delete this? Or did you just copy my gist into `docs/bootstrap/README.md`?

Reply:
Correct, I have only updated the faasd version to `0.8.1` and copied it to `docs/bootstrap/README.md`.
Is that ok, or would you prefer to keep it as a gist?

Reply:
OK thanks for explaining